a3p-integration/proposals/z:acceptance/test-lib/kread.js

```diff
@@ -344,3 +337,19 @@ export const getBalanceFromPurse = async (address, type) => {
 
   return assetPurse?.balance.value.payload[0]?.[0] || null;
 };
+
+export const getAssetAmount = (brand, asset) => {
+  const assetValue = makeCopyBag([[asset, 1n]]);
+  const assetAmount = AmountMath.make(
+    brand,
+    //@ts-expect-error casting
```

Comment:
This shouldn't be necessary. `AmountMath.make` takes CopyBagAmounts by default, so you should be able to declare a type for `assetValue` and not have to expect an error. The following worked for me

```
/** @import {Brand, CopyBagAmount} from '@agoric/ertp'; */

/**
 * @param {Brand} brand
 * @param {any} asset
 * @returns {CopyBagAmount<'item'>}
 */
export const getAssetAmount = (brand, asset) => {
  const assetValue = makeCopyBag([[asset, 1n]]);
  return AmountMath.make(brand, assetValue);
};
```

Reply:
Requested change addressed